src/contexts/Localisation.tsx

```diff
@@ -44,26 +47,34 @@ interface LanguageState {
   error: any
 }
 
+type Substitutions = Record<string, unknown> | string | number
+
+export type TranslateMethod = (key: keyof typeof strings, substitutions?: Substitutions) => string
+
+type TranslatePluginMethod = (key: string, substitutions?: Substitutions) => string
+
 const initialContext: {
-  strings: LanguageStrings
   selectedLanguage: LanguageOption
   languageOptions: LanguageOption[]
   languageOptionsFull: LanguageOption[]
   loading: boolean
   error: any
   setLanguage: Function
-  getPluginStrings: Function
   refetchLanguages: Function
+  translate: TranslateMethod
+  t: TranslateMethod
+  getPluginTranslator: (pluginCode: string) => TranslatePluginMethod
 } = {
-  strings: {} as LanguageStrings,
   selectedLanguage: initSelectedLanguage,
   languageOptions: [],
   languageOptionsFull: [],
   loading: true,
   error: null,
   setLanguage: () => {},
-  getPluginStrings: () => {},
   refetchLanguages: () => {},
+  translate: () => '',
+  t: () => '',
```

Comment:
This is so we can use either the shortcut name `t`, or the full name `translate` when calling. Mostly we'll use `t`, but there might be the odd occasion where the more verbose name is helpful.